ml-proto/host/builtins.ml

```diff
@@ -1,9 +1,50 @@
 open Source
+open Eval
+open Memory
+open Types
```

Comment:
I assume it's just: if you're not using any of the things in the module, you don't need the open. I was hoping it would be just a better of deleting the open, seeing if everything still compiles, and if so, go with it.